modular_citadel/code/modules/custom_loadout/custom_items.dm

```diff
@@ -439,3 +436,24 @@
 	icon_state = "moveralls"
 	mutantrace_variation = NO_MUTANTRACE_VARIATION
 
+/obj/item/clothing/head/telegramhat
+	name = "Singing Telegram Hat"
+	icon = 'icons/obj/custom.dmi'
+	alternate_worn_icon = 'icons/mob/custom_w.dmi'
```

Comment:
idem.
Your worn icon states are actually located in a rather outdated version of `modular_citadel/icons/obj/clothing/turtlenecks.dmi` it seems, they should be moved elsewhere. And make sure the file is up to date with the upstream.

Reply:
Its a turtleneck so I added it their